README.md

```diff
@@ -1,2 +1,99 @@
 # open-sir
-Python SIR-x model implementation
+
+Open-SIR is an Open Source Python project for modelling pandemics and infectious diseases using Compartmental Models, such as the widely used [Susceptible-Infected-Removed (SIR) model](http://rocs.hu-berlin.de/corona/docs/forecast/model/#classic-sir-dynamics). 
+
+The current stage of the software is *Alpha*.
+
+## Features
+- Model the dynamics of infectious diseases
+- Parameter fitting
+- Calculation of confidence intervals
+- CLI for interfacing with non Python environments (Bash, Node.JS, Matlab, etc).
+
+So far, Open-SIR provides an implementation of the SIR model and the recently new SIR based model SIR-X, developed by the [Robert Koch Institut](http://rocs.hu-berlin.de/corona/docs/forecast/model/#sir-x-dynamics-outbreaks-with-temporally-increasing-interventions).
```

Comment:
@jia200x @felipehuerta17 what do you think about eventually adding a new section here that lists the "upcoming" features?

I guess it is very soon to add it now, but maybe next week once we have those more clear?

Reply:
+1!

Reply:
Done. Let's include it now as it can attract interest!
We can "commit" to at least two approaches in the future

* Incorporate SEIR model
* Advanced methods to estimate reproduction number